adr/ADR-8.md

```diff
@@ -360,15 +385,91 @@ first time any message has a `Pending==0`. This signal must also be sent if no d
 is clearly indicated as a language specific extension. Names should be language idiomatic but close to these below.
 
-|Name|Description|
-|----|-----------|
-|`IncludeHistory`|Send all available history rather than just the latest entries|
-|`IgnoreDeletes`|Only sends `PUT` operation entries|
-|`MetaOnly`|Does not send any values, only metadata about those values|
-|`UpdatesOnly`|Only sends new updates made, no current or historical values are sent. The End Of Initial Data marker is sent as soon as the watch starts.|
+| Name             | Description                                                                                                                                |
+|------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| `IncludeHistory` | Send all available history rather than just the latest entries                                                                             |
+| `IgnoreDeletes`  | Only sends `PUT` operation entries                                                                                                         |
+| `MetaOnly`       | Does not send any values, only metadata about those values                                                                                 |
+| `UpdatesOnly`    | Only sends new updates made, no current or historical values are sent. The End Of Initial Data marker is sent as soon as the watch starts. |
 
 The default behavior with no options set is to send all the `last_per_subject` values, including delete/purge operations.
 
+#### Replicas for NATS Server 2.10 and newer
+
+Since NATS Server 2.10 we support transforming messages as a stream configuration item. This allows us to source one 
+bucket from another and rewrite the keys in the new bucket to have the correct name.
+
+To copy the keys `NEW.>` from bucket `ORDERS` into `NEW_ORDERS` we create the new stream with the following 
+partial config:
+
+```json
+    "sources": [
+      {
+        "name": "KV_ORDERS",
+        "filter_subject": "$KV.ORDERS.NEW.>",
+        "subject_transforms": [
+          {
+            "src": "$KV.ORDERS.>",
+            "dest": "$KV.NEW_ORDERS.>"
+          }
+        ]
+      }
+    ],
+```
+
+This results in all messages from `ORDERS` keys `NEW.>` to be copied into `NEW_ORDERS` and the subjects rewritten on 
+write to the new bucket so that a unmodified KV client on `NEW_ORDERS` would just work.
+
+As this is a `Source` and not a `Mirror` this new bucket can accept writes.  Sourced streams can be created with no
+listening subjects which would render the mirror a read-only replica.  Given that it's a Source it can be made with
+multiple sources to create an aggregate bucket by using multiple subject transforms.
+
+#### Read replica mirrors
+
+Regional read replicas can be built using the standard mirror feature by setting `mirror_direct` to true as long as the
```

Comment:
`Regional` in context of NATS naming convention is not clear.

Reply:
I dont think we have a good term :) these are typically a geo distributed super cluster and the purpose is to have network-wise near replicas for reading.  What would you suggest?